CONTRIBUTING.md

```diff
@@ -1,39 +1,33 @@
 Contributing
 ============
 
-Thank you for considering contributing to Mastodon 🐘
+Thank you for considering contributing to Florence Mastodon
 
 You can contribute in the following ways:
 
 - Finding and reporting bugs
 - Translating the Mastodon interface into various languages
 - Contributing code to Mastodon by fixing bugs or implementing features
 - Improving the documentation
+- Joining the conversation on our [Chat]
+- Participate in OutReach or other activities
+- Proof-read
+- And much more
 
-If your contributions are accepted into Mastodon, you can request to be paid through [our OpenCollective](https://opencollective.com/mastodon).
+If you want to contribute with monetary support you can do so via our [Open Collective](https://opencollective.com/florence-social) (which isn't publicly launched yet)
```

Comment:
If this isn't publicly-launched, should we be linking to it yet? Is there a temporary stop-gap solution or time-frame for this?